### ISSUE TITLE
preset-create-react-app@3.1.0 breaks props doc gen

### ISSUE THREAD

**Issue by luistorres** [storybookjs/presets#150]
Steps to reproduce:
- Just follow the steps [here](https://gist.github.com/shilman/bc9cbedb2a7efb5ec6710337cbd20c0c) (new CRA app & setup storybook)

Result when running storybook:
```
info @storybook/react v5.3.19
info
info => Loading static files from: /Users/user/projects/cra-ts-storybook/public .
info => Loading presets
WARN   Failed to load preset: "@storybook/preset-create-react-app" on level 1
ERR! Error: Cannot find module 'react-docgen-typescript-plugin'
ERR!     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
ERR!     at Function.Module._load (internal/modules/cjs/loader.js:506:25)
ERR!     at Module.require (internal/modules/cjs/loader.js:636:17)
ERR!     at require (internal/modules/cjs/helpers.js:20:18)
ERR!     at Object.<anonymous> (/Users/user/projects/cra-ts-storybook/node_modules/@storybook/preset-create-react-app/dist/index.js:41:56)
ERR!     at Module._compile (internal/modules/cjs/loader.js:688:30)
ERR!     at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
ERR!     at Module.load (internal/modules/cjs/loader.js:598:32)
ERR!     at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
ERR!     at Function.Module._load (internal/modules/cjs/loader.js:529:3)
ERR!     at Module.require (internal/modules/cjs/loader.js:636:17)
ERR!     at require (internal/modules/cjs/helpers.js:20:18)
ERR!     at interopRequireDefault (/Users/user/projects/cra-ts-storybook/node_modules/@storybook/core/dist/server/presets.js:164:18)
ERR!     at loadPreset (/Users/user/projects/cra-ts-storybook/node_modules/@storybook/core/dist/server/presets.js:174:22)
ERR!     at presets.reduce (/Users/user/projects/cra-ts-storybook/node_modules/@storybook/core/dist/server/presets.js:230:20)
ERR!     at Array.reduce (<anonymous>)
ERR!  { Error: Cannot find module 'react-docgen-typescript-plugin'
ERR!     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
ERR!     at Function.Module._load (internal/modules/cjs/loader.js:506:25)
ERR!     at Module.require (internal/modules/cjs/loader.js:636:17)
ERR!     at require (internal/modules/cjs/helpers.js:20:18)
ERR!     at Object.<anonymous> (/Users/user/projects/cra-ts-storybook/node_modules/@storybook/preset-create-react-app/dist/index.js:41:56)
ERR!     at Module._compile (internal/modules/cjs/loader.js:688:30)
ERR!     at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
ERR!     at Module.load (internal/modules/cjs/loader.js:598:32)
ERR!     at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
ERR!     at Function.Module._load (internal/modules/cjs/loader.js:529:3)
ERR!     at Module.require (internal/modules/cjs/loader.js:636:17)
ERR!     at require (internal/modules/cjs/helpers.js:20:18)
ERR!     at interopRequireDefault (/Users/user/projects/cra-ts-storybook/node_modules/@storybook/core/dist/server/presets.js:164:18)
ERR!     at loadPreset (/Users/user/projects/cra-ts-storybook/node_modules/@storybook/core/dist/server/presets.js:174:22)
ERR!     at presets.reduce (/Users/user/projects/cra-ts-storybook/node_modules/@storybook/core/dist/server/presets.js:230:20)
ERR!     at Array.reduce (<anonymous>)
ERR!   stack:
ERR!    'Error: Cannot find module \'react-docgen-typescript-plugin\'\n    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)\n    at Function.Module._load (internal/modules/cjs/loader.js:506:25)\n    at Module.require (internal/modules/cjs/loader.js:636:17)\n    at require (internal/modules/cjs/helpers.js:20:18)\n    at Object.<anonymous> (/Users/user/projects/cra-ts-storybook/node_modules/@storybook/preset-create-react-app/dist/index.js:41:56)\n    at Module._compile (internal/modules/cjs/loader.js:688:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)\n    at Module.load (internal/modules/cjs/loader.js:598:32)\n    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)\n    at Function.Module._load (internal/modules/cjs/loader.js:529:3)\n    at Module.require (internal/modules/cjs/loader.js:636:17)\n    at require (internal/modules/cjs/helpers.js:20:18)\n    at interopRequireDefault (/Users/user/projects/cra-ts-storybook/node_modules/@storybook/core/dist/server/presets.js:164:18)\n    at loadPreset (/Users/user/projects/cra-ts-storybook/node_modules/@storybook/core/dist/server/presets.js:174:22)\n    at presets.reduce (/Users/user/projects/cra-ts-storybook/node_modules/@storybook/core/dist/server/presets.js:230:20)\n    at Array.reduce (<anonymous>)',
ERR!   code: 'MODULE_NOT_FOUND'
```

**Comment by robotboy** [comment]
hi @luistorres, I had the same issue just today. Adding `react-docgen-typescript-plugin` to the project solves this error.

**Comment by shilman** [comment]
Cc @mrmckeb

**Comment by shilman** [comment]
Fixed (hopefully) in 3.1.1
